virtual-host-gatherer/virtual-host-gatherer.spec

```diff
@@ -39,8 +43,8 @@ BuildRequires:  libxslt-tools
 BuildRequires:  %{python_module devel}
 BuildRequires:  %{python_module pycurl}
 BuildRequires:  %{python_module six}
-Requires:       python-pycurl
-Requires:       python-six
+Requires:       %{python_module pycurl}
+Requires:       %{python_module six}
```

Comment:
This scares me most. As by docs this is wrong. but without it was not working.
I think I need to re-test this.

Reply:
Right, that's not nice. Would re-using the same python-rpm-macros rpm be a viable option that avoids these workarounds?

Reply:
This also happens on Leap. We need to investigate why, it shouldn't happen. But it's unrelated to using `python-rpm-macros` from AlmaLinux:

```sh
➤ osc buildinfo systemsmanagement:Uyuni:Master virtual-host-gatherer openSUSE_Leap_15.4 x86_64 | rg python-rpm-macros
  <bdep name="python-rpm-macros" version="20220809.cf8a7b8" release="150400.3.3.1" arch="noarch" hdrmd5="291474f1456153adc85234036945c81b" project="SUSE:SLE-15-SP4:Update" repository="pool"/>
```

Let's keep it these changes for now.